### ISSUE TITLE
Node and Validator Epic

### ISSUE THREAD

**Issue by icerove** [near/near-explorer#549]
### Give three table and nav tags on the table for validators, proposals and online-nodes

####node dashboard term will include(with fake data):

Current epoch progress: 90.14%
Time until new epoch 00:21:52
Previous epoch length: 04:19:34
Epoch ID: 9vUWGQnUeDozaiH7L8Vs3ptphg1gtgZTfZoVsRQRto8c
stake total amount: 123M
stake APY : 16%
Circulating Supply: 123M
seat price: 66,455 N

See #370 for the initial conversation

**Comment by icerove** [comment]
@frol @corwinharrell want to put the design for node part in this new epic since it is not only validator part and after redesign, many issues will be solved at the same time

**Comment by frol** [comment]
Sounds good to me!

**Comment by icerove** [comment]
I review again and find out several things for design:
1. epoch APY if necessary 
![image](https://user-images.githubusercontent.com/33007476/110521943-3c173a80-80c5-11eb-8449-22ba4b606c76.png)
2. circulating supply only show supply 
![image](https://user-images.githubusercontent.com/33007476/110522041-551feb80-80c5-11eb-802b-8795e9e485fc.png)
3. NEAR price in dollar ???
4. for validator, we have three status: 
      -1. normal one,
      -2. upcoming one,
![image](https://user-images.githubusercontent.com/33007476/110522497-cfe90680-80c5-11eb-9c29-f019a080680b.png)

      -3. next epoch kick off one
![image](https://user-images.githubusercontent.com/33007476/110522457-c495db00-80c5-11eb-9cde-94036ff40125.png)

currently we use different color of icon to show and when hover, there will be explanation, should we have better way to do so?
see this: 
![image](https://user-images.githubusercontent.com/33007476/110522411-b8aa1900-80c5-11eb-8615-88a5c34cf922.png)
5. online epoch number for each validator ???
6. this banner
![image](https://user-images.githubusercontent.com/33007476/110523369-de83ed80-80c6-11eb-9579-be43c17a3cdb.png)

@corwinharrell

**Comment by corwinharrell** [comment]
@icerove in response:

1. Is this necessary? How much of a value add is this? cc @jimmy3dita 
2. IMO it benefits us to show the circulating supply as as fraction of the total supply. It at least doesn't hurt. Any reason why we shouldn't include it?
3. I've asked this before, and I don't think we have an accurate API to source the fiat conversion. cc @kcole16
4. It's not exactly clear to me what these statuses mean. The language is pretty unclear. Could I get some clearer explanations for each status? cc @jimmy3dita @frol 
5. Not sure what this means, could you clarify?
6. Yes I will add this divider back into the design

**Comment by icerove** [comment]
2. follow your step with design
5. ignore this

**Comment by jimmy3dita** [comment]
1. may have implications on "promising future value", so I'd skip that one
4. Denisk (the creator of near-staking) condensed the validator status information as follows:
- red dot means the node missed the minimum requirements for stake and/or blocks missed (>10%)
- yellow dot means the node missed more than 1% of the blocks already
- `next: kicked out` means this very specific node will be kicked out in the next epoch

I personally don't see 1 nor 4 as key features.

```
Current epoch progress: 90.14%
Time until new epoch 00:21:52
Previous epoch length: 04:19:34
Epoch ID: 9vUWGQnUeDozaiH7L8Vs3ptphg1gtgZTfZoVsRQRto8c
stake total amount: 123M
stake APY : 16%
Circulating Supply: 123M
seat price: 66,455 N
```

I would replace `Epoch ID: 9vUWGQnUeDozaiH7L8Vs3ptphg1gtgZTfZoVsRQRto8c` with `Epoch started at block #31905134"

**Comment by corwinharrell** [comment]
@icerove @frol @jimmy3dita The Figma for the validator dashboard is up to date with all the most recent changes:

https://www.figma.com/file/CGGQaZEXTxl37ItJW6DlDD/Validator-Dashboard?node-id=0%3A1

I wanted to let you know that I also added a section in the document with spec for a new table component. Once built, we should be able to leverage this component across the app and to expand it over time with additional functionality if necessary.

Since the tables on this page are so dense, I'm not 100% sure on how we should handle the table on mobile. I'll think on this a bit more and let you know if I can figure anything out that works better than a horizontal scroll given how many cells each row contains.

If there are any questions let me know but this shouldn't be too different from before albeit a couple data additions.

**Comment by icerove** [comment]
great work!
since mobile space is limited, we definitely can delete some detail information there, no worries about that
@corwinharrell

**Comment by frol** [comment]
@corwinharrell this looks great! By the way, @shelegdmitriy has already started implementing this redesign

**Comment by frol** [comment]
Implemented in #661
